src/AppKit/Compat.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.ComponentModel;
+
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.AppKit{
+#if !XAMCORE_4_0
+	public partial class NSCollectionViewTransitionLayout {
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("init")]
+		public NSCollectionViewTransitionLayout () : base (NSObjectFlag.Empty)
```

Comment:
Sorry but it remains unclear why you're removing it from bindings. The two main reasons are:

1. It was our mistake and it does not work (the normal case for `[Obsolete]`). There's no point to have the generated code inside the body, it will make it harder to diagnose if called;

2. It was Apple mistake and they removed it. Here it should be `[Obsoleted]` or `[Deprecated]` (and can be kept part of the bindings).